app/mailers/school_mailer.rb

```diff
@@ -228,17 +228,13 @@ def induction_coordinator_reminder_to_choose_route_email(recipient:, name:, scho
     )
   end
 
-  def induction_coordinator_reminder_to_choose_provider_email(recipient:, name:, school_name:, sign_in_url:)
+  def induction_coordinator_reminder_to_choose_provider_email(recipient:)
     template_mail(
       COORDINATOR_REMINDER_TO_CHOOSE_PROVIDER_EMAIL_TEMPLATE,
       to: recipient,
       rails_mailer: mailer_name,
       rails_mail_template: action_name,
-      personalisation: {
-        name: name,
-        school_name: school_name,
-        sign_in_url: sign_in_url,
-      },
+      personalisation: {},
```

Comment:
We don't have any scheduled emails that will break if we remove the template variables, right?